app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < Devise::SessionsController
+  respond_to :json
+
+  def create
+    resource = User.find_for_database_authentication(email: params[:user][:email])
+    return invalid_login_attempt unless resource
+
+    if resource.valid_password?(params[:user][:password])
+      sign_in :user, resource
+      return render nothing: true
```

Comment:
lets change this to `render json: {}, status: :ok` so we are consistent with registrations